packages/labs/src/components/select/multiSelect.tsx

```diff
@@ -62,8 +62,8 @@ export interface IMultiSelectProps<T> extends IListItemsProps<T> {
     /** Props to spread to `TagInput`. */
     tagInputProps?: Partial<ITagInputProps> & object;
 
-    /** Custom renderer to transform an item into a string for tags. */
-    tagRenderer: (item: T) => string;
+    /** Custom renderer to transform an item into an element for tags. */
```

Comment:
"into [tag content]."

ReactNode is decidedly _not_ "an element," as it can also be an array and can be non-element things like string or number.